keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -506,6 +507,29 @@ With recent conversations about kube-apiserver and enabled featureset version,
 it is possible to incorporate other version relating to the cluster, such as
 minimum kubelet version, maximum kubelet version, and enabled featureset version.
 
+#### Credentials
+
+An array of credentials to access the cluster. Each credential could be assigned 
+to a different consumer. Each credential has: 
+- a **consumer** field to indicate which consumer it is assigned to. 
+- an **accessRef** field to point to the access information location. 
+
+The access information itself is stored in the same Kubernetes cluster on which
+the ClusterProfile API resides. However, the access information for different 
+consumers is stored in different namespaces to ensure that it is only accessible
+to the consumer for which it is intended. We recommend that the access information
+be stored in a secret object to add further protection to the access information.
+
+##### Access information format
+The access information must contain the following fields
+- **Config**: This field contains cluster access information compatible with the 
+  [kubeconfig format](https://github.com/kubernetes/kubernetes/blob/dbc2b0a5c7acc349ea71a14e49913661eaf708d2/staging/src/k8s.io/client-go/tools/clientcmd/api/types.go#L31).
+
+Please note that since a single [Kubeconfig file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+supports access to multiple clusters. Consequently, multiple `ClusterProfile`
```

Comment:
```suggestion
Please note that a single [Kubeconfig file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
supports access to multiple clusters. Consequently, multiple `ClusterProfile`
```

(or keep “since” and merge the two sentences — but I think separate sentences is better).

In a scenario with multiple clusters (or contexts) in a single Kubeconfig entry, how is the appropriate set of credentials chosen? Should there be an optional context field?

Reply:
I added a context field